surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+# Survey questions
+
+Whether or not you contribute to the Rust Project <https://github.com/rust-lang>
+today, we want to hear from you!
+
+The Rust Foundation has created this survey to help us identify the main
+problems negatively affecting active, past, and potential contributors to the
+Rust Project to help us plan programs that can most effectively support the
+contributors and maintainers of the Rust Project.
+
+Unless you choose to enter your email, your answers will be anonymous. Any
+personal data you submit as a part of this survey will be handled in accordance
+with our policy as described in our Frequently Asked Questions:
+
+https://github.com/rust-community/team/wiki/State-of-the-Rust-Language-Community-Survey-FAQ
+
+We estimate it will take about ? minutes to complete.
+
+> TODOs:
+>
+> - Evaluate survey length (the interviews took ~20-30 minutes)
+> - Can the community survey FAQ be generalized for all surveys such as this or
+>   the trademark surveys?
+> - Resolve and close open issues related to this survey on `rust-lang/surveys`
+> - Are these questions being presented in the optimal order?
+> - Make sure none of the major cohort sections are leaving out relevant
+>   questions that are asked of the other major cohorts.
+
+## Collaboration Interview
+
+### Have you contributed or ever considered contributing to the official Rust project (e.g. any repo that is part of the rust-lang Github org)?
```

Comment:
nit: ~~Github~~ => GitHub

Reply:
We should define "contribution". Do we care about consistency of contribution?

Reply:
> Do we care about consistency of contribution?

What do you mean?

Reply:
Nevermind, we address what I meant with other questions.